From 438f3391d15b0ad864ace233e9eca784e9b52a34 Mon Sep 17 00:00:00 2001
From: Nathan Muggli <nmuggli@google.com>
Date: Mon, 26 Aug 2024 21:42:14 -0600
Subject: [PATCH] Add message_pump_libevent

This patch reverts the removal of this header file only - it does not
revert the whole commit.

Change-Id: I533a2a0dc05a894797719764db7d1da38b4564c4
---
 base/message_loop/message_pump_libevent.h | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)
 create mode 100644 base/message_loop/message_pump_libevent.h

diff --git a/base/message_loop/message_pump_libevent.h b/base/message_loop/message_pump_libevent.h
new file mode 100644
index 0000000000..eabcbaf7c4
--- /dev/null
+++ b/base/message_loop/message_pump_libevent.h
@@ -0,0 +1,18 @@
+// Copyright 2012 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef BASE_MESSAGE_LOOP_MESSAGE_PUMP_LIBEVENT_H_
+#define BASE_MESSAGE_LOOP_MESSAGE_PUMP_LIBEVENT_H_
+
+#include "base/message_loop/message_pump_epoll.h"
+
+namespace base {
+
+// TODO(crbug.com/40057013): Rewrite all references to MessagePumpLibevent and
+// delete this header.
+using MessagePumpLibevent = MessagePumpEpoll;
+
+}  // namespace base
+
+#endif  // BASE_MESSAGE_LOOP_MESSAGE_PUMP_LIBEVENT_H_
-- 
2.46.0.295.g3b9ea8a38a-goog

